tests/openvino/test_modeling.py

```diff
@@ -521,10 +521,11 @@ class OVModelForCausalLMIntegrationTest(unittest.TestCase):
         "stablelm",
         "starcoder2",
         "phi",
+        "internlm2",
```

Comment:
would be nice to also add :

```suggestion
        "internlm2",
        "decilm",
        "olmo",
        "orion",
```

Reply:
I plan to add it after spliting tests on nightly to not increase testing time too much

Reply:
for some models I have difficulties with creating valid dummy models, do not block merging by this issue, I'll remove these models support

Reply:
thanks @eaidova